specification/2.0/README.md

```diff
@@ -2183,38 +2150,65 @@ Application-specific data.
 <a name="reference-material"></a>
 ## material
 
-The material appearance of a primitive.
+glTF 2.0 uses the metal-roughness material model from Physically Based Renderring. The metal-roughness material model is defined by the following properties:
```

Comment:
Property reference section will be auto-generated from JSON-Schemas. This brief description should go into Materials (above) part of README, while exact equations and implementation details should be in a new Appendix.